### PR TITLE
media-fonts/fonts-meta: add new fonts-meta package

### PR DESCRIPTION
**Pull request by juippis**
- this is to replace the infinality-ultimate-meta package. Please remove the
   infinality-related packages in case you had it installed,
     app-eselect/eselect-infinality
     app-eselect/eselect-lcdfilter
     media-fonts/infinality-ultimate-meta
     media-libs/fontconfig-ultimate
     media-libs/fontconfig-infinality
   and see https://wiki.gentoo.org/wiki/Fontconfig#Enabling_Infinality for
   instructions how to "undo" the infinality font settings in your system.
   Remember to update your infinality-ultimate-meta's package.use atom to
   match media-fonts/fonts-meta now.

### CODE REVIEW

**Comment by juippis** [comment]
```diff
--- /var/db/repos/gentoo/media-fonts/infinality-ultimate-meta/infinality-ultimate-meta-1.ebuild	2021-03-22 14:48:43.907570143 +0200
+++ ./fonts-meta-1.ebuild	2021-03-29 15:53:16.649029951 +0300
@@ -3,13 +3,13 @@
 
 EAPI=7
 
-DESCRIPTION="Meta package for infinality-ultimate with fonts"
+DESCRIPTION="Meta package for fonts to cover most needs"
 HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
 
 LICENSE="metapackage"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="+free +latin minimal ms"
+IUSE="minimal +latin +free ms"
 
 LANGS="as bn bo brx doi dz gu hi ja km kn ko kok ks mai ml mr ne or pa
 	sa sat sd si syc ta te zh"
@@ -18,8 +18,7 @@
 done
 unset lang LANGS
 
-RDEPEND="
-	media-libs/fontconfig-ultimate
+RDEPEND="media-libs/fontconfig
 	free? (
 		media-fonts/noto
 		media-fonts/heuristica
@@ -28,6 +27,7 @@
 		latin? (
 			media-fonts/courier-prime
 			media-fonts/dejavu
+			media-fonts/quivira
 			media-fonts/signika
 			media-fonts/tex-gyre
 			!minimal? (
@@ -35,6 +35,8 @@
 				media-fonts/droid
 				media-fonts/font-bh-ttf
 				media-fonts/font-cursor-misc
+				media-fonts/hack
+				media-fonts/ibm-plex
 				media-fonts/open-sans
 				media-fonts/source-pro
 				media-fonts/ubuntu-font-family
@@ -93,4 +95,5 @@
 			media-fonts/open-sans
 			media-fonts/paratype
 		)
-	)"
+	)
+	!media-fonts/infinality-ultimate-meta"
```
Suggestions welcome!

File: media-fonts/fonts-meta/fonts-meta-1.ebuild [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+EAPI=7
+
+DESCRIPTION="Meta package for fonts to cover most needs"
+HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
+
+LICENSE="metapackage"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="minimal +latin +free ms"
```
Comment (juippis) [inline media-fonts/fonts-meta/fonts-meta-1.ebuild:12]: Apparently in the older copy of infinality-ultimate-meta these weren't sorted!

File: media-fonts/fonts-meta/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>juippis@gentoo.org</email>
+    <name>Fonts</name>
+  </maintainer>
+  <use>
+    <flag name="free">Install fonts for the "free" preset</flag>
```
Comment (DarthGandalf) [inline media-fonts/fonts-meta/metadata.xml:9]: What are these "free" and "ms" presets?

Comment (juippis) [reply media-fonts/fonts-meta/metadata.xml]: No clue, probably some old language not so relevant anymore.
https://hub.mpcthemes.net/knowledgebase/typography-preset/

What do you suggest as the replacement?

Comment (DarthGandalf) [reply media-fonts/fonts-meta/metadata.xml:17]: I'm not sure how that website's presets are applicable here...

If I knew what these meant, I'd provide some suggestion :)

**Comment by joakim-tjernlund** [comment]
Assuming USE=ms is about Microsoft compatible fonts you want:
media-fonts/croscorefonts
media-fonts/crosextrafonts-caladea
media-fonts/crosextrafonts-carlito

Libreoffice will look so much better with the last 2 fonts

**Comment by joakim-tjernlund** [comment]
media-fonts/noto and media-fonts/noto-emoji probably, not sure which USE the belong too though

**Comment by juippis** [comment]
@joakim-tjernlund good idea, I'll add them, thanks!

**Comment by juippis** [comment]
@joakim-tjernlund not sure if you receive any e-mails from the fork pings so I'll ping here...
Github should've fixed the issue and you (or anyone in future) shouldn't receive pings from that scenario anymore. Well, not that I'm going to repeat the mistake, but I'd like to believe something good came out of it, for everyone.

**Comment by joakim-tjernlund** [comment]
@juippis , I don't get emails any more. Thanks a lot :)

**Pull request closed without merging**